test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -82,27 +80,10 @@ public void run() {
 }
 
 class testFrame extends JFrame implements ActionListener {
-    final Map<String, String> lookAndFeelMap = new HashMap<>();
     public testFrame() {
```

Comment:
Class names should start with a capital letter.

Reply:
Done.